src/backends/tensorrt/caffe.proto

```diff
@@ -0,0 +1,2235 @@
+syntax = "proto2";
```

Comment:
I understand why we need this, and that's a copy of our Caffe protofile. Can we download it with cmake instead ? Reason is that although it may not change too much anymore, downloading it would avoid maintaining this file when changes do occur.

A download link to the up to date version should be this: https://raw.githubusercontent.com/jolibrain/caffe/master/src/caffe/proto/caffe.proto

Reply:
ok, done